client/utils/loadComponent.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+import Loadable from 'react-loadable';
```

Comment:
We don't need to make use of an extra package to achieve Route level codesplitting.

As I shared with you, there's a simpler way to do it with an HOC. We should only use NPM packages when it's necessary.